data.sql

```diff
@@ -1,5 +1,23 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Agumon', '2020-03-02', 0, true, 10.23);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Gabumon', '2018-11-15', 2, true, 8.0);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Devimon', '2017-05-12', 5, true, 11.0);
+
```

Comment:
- [ ] It would be better if we insert the data as seen in the screenshot below, it is the only data missing
![Screenshot from 2023-08-14 06-03-13](https://github.com/aamir-asaram/vet-clinic/assets/46388113/a4c0b18f-d4b4-4c26-92aa-8a5e4cb73f76)